util/reggen/ip_block.py

```diff
@@ -56,7 +56,8 @@
     32: 'alert_handler',
     33: 'rv_plic',
     34: 'ast',
-    35: 'sensor_ctrl'
+    35: 'sensor_ctrl',
+    40: 'ascon'
```

Comment:
The reason for leaving a gap here is that IDs 36-39 are already taken by IPs that have been added on `integrated_dev`. I'll create a separate PR to fill this gap on `master`, so that we don't accidentally create collisions that will make reconvergence more difficult.